.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 rust/**/target/
-js/soluss
+js/soluss
```

Comment:
here a newline is missing at the end of the file, maybe we could put it back, this also will prevent for seeing a change in the `.gitignore` while actually nothing changed.